### ISSUE TITLE
No uptime when a libvirt group user operate the system connection VM

### ISSUE THREAD

**Issue by yunmingyang** [cockpit-project/cockpit-machines#1871]
Description of problem:
When a libvirt group user try to operate the system connection VM, there will be no uptime shown on the pop-up dialog

Version-Release number of selected components (if applicable):
cockpit-322-1.fc40.x86_64
cockpit-machines-316-1.fc40.noarch
libvirt-dbus-1.4.1-4.fc40.x86_64

How reproducible:
100%

Steps to Reproduce:
1 Create a libvirt group user, then create a system connection VM
2 Log in with the libvirt group user, then try to operate the system connection VM

Actual results:
1 No uptime shown on the dialog after the step 2

Expected results:
1 There should be the uptime shown on the dialog or a hint for clarifying

Additional info:
It seems that libvirt group users may not have enough privilege for accessing the log in /var/log/libvirt/qemu

**Comment by yunmingyang** [comment]
May be related with the failures of https://github.com/cockpit-project/cockpit-machines/pull/1853

**Comment by mvollmer** [comment]
> It seems that libvirt group users may not have enough privilege for accessing the log in /var/log/libvirt/qemu

Yes, that seems to be the reason. We should be able to get the start time of the qemu process from /proc, I think.
